src/components/textField/index.tsx

```diff
@@ -208,7 +208,12 @@ const TextField = (props: InternalTextFieldProps) => {
             </View>
           )}
           {showClearButton && (
-            <ClearButton onClear={onClear} testID={`${props.testID}.clearButton`} onChangeText={onChangeText}/>
+            <ClearButton
+              onClear={onClear}
+              testID={`${props.testID}.clearButton`}
+              onChangeText={onChangeText}
+              preset={props.preset}
```

Comment:
The ClearButton component shouldn't be aware of the preset. I think we should pass 'style' instead

Reply:
Fixed